app/controllers/v1/statuses_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module V1
+  # API for Compliance Status
+  class StatusesController < ApplicationController
+    skip_around_action :authenticate_user
+
+    ARB = ActiveRecord::Base
+
+    def show
+      render json: { data: all_statuses }, status: status_return_code
+    end
+
+    private
+
+    def all_statuses
+      @all_statuses ||= {
+        api: api_status
```

Comment:
Do you think `api` is OK? Or, maybe we should just say `db` here?

Reply:
I guess `api` is fine for now.